packages/xstate-vue/src/xstate-machine.ts

```diff
@@ -0,0 +1,103 @@
+import { interpret, State } from 'xstate';
+import Vue, { VNode } from 'vue';
+
+export default Vue.extend({
+  render(h): VNode {
+    const { current, service, send } = this;
+    return h('div', [
+      this.$scopedSlots.default({
+        current,
+        service,
+        send
+      })
+    ]);
+  },
+  props: {
+    machine: {
+      type: Object,
+      required: true,
+      default: () => ({})
+    },
+    options: {
+      type: Object,
+      default: () => ({})
+    }
+  },
+  data() {
+    return {
+      current: null
+    };
+  },
+  computed: {
+    initialState() {
+      if (!this.service.initialState) return;
+      return this.service.initialState;
+    },
+    actions() {
+      return this.options.actions;
+    },
+    services() {
+      return this.options.services;
+    }
+  },
+  watch: {
```

Comment:
This is not needed anymore. See @davidkpiano's [comment](https://github.com/davidkpiano/xstate/pull/864#issuecomment-569761475).